XamlAnimatedGif.Shared/Animator.cs

```diff
@@ -546,6 +548,12 @@ private static Task<Stream> GetStreamFromUriAsync(Uri uri)
                 }
                 throw new Exception("Resource not found");
             }
+
+            if (uri.Scheme.StartsWith("http"))
```

Comment:
Is it enough? If someone has the stupid idea of passing an URL like "httpfoobar://", it will fail miserably ;)
Of course, it will fail anyway if we don't accept it, but at least the error message will be more explicit. I think it would be better to check explicitly for "http" and "https".